### ISSUE TITLE
[EN]{Event] Children's Week 2017

### ISSUE THREAD

**Issue by monibu** [Atlantiss/BugTracker#9129]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Event quest are barely working

**Current behaviour**: Stormwind/Orgrimmar orphan quest should be repeatable each year. Currently they are unavailable if someone did the in previous years.
For the Alliance the quests are:
1.http://www.wowhead.com/quest=1468/childrens-week
2a.http://www.wowhead.com/quest=1687/spooky-lighthouse
2b.http://www.wowhead.com/quest=1558/the-stonewrought-dam
2c.http://www.wowhead.com/quest=1479/the-bough-of-the-eternals
3a.http://www.wowhead.com/quest=558/jainas-autograph
3b.http://www.wowhead.com/quest=915/you-scream-i-scream
4.http://www.wowhead.com/quest=171/a-warden-of-the-alliance
And for the horde:
1.http://www.wowhead.com/quest=172/childrens-week
2a.http://www.wowhead.com/quest=910/down-at-the-docks
2b.http://www.wowhead.com/quest=911/gateway-to-the-frontier
2c.http://www.wowhead.com/quest=1800/lordaeron-throne-room
3a.http://www.wowhead.com/quest=925/cairnes-hoofprint
3b.http://www.wowhead.com/quest=915/you-scream-i-scream
4.http://www.wowhead.com/quest=5502/a-warden-of-the-horde
Or for a quick database update: 171, 172, 558, 910, 911, 915, 925, 1479, 1468, 1558, 1687, 1800, 5502

Outland and Northrend quest are not scripted. You can get them, but you can't complete a single one of them.
Ally Shatt quests:
1.http://www.wowhead.com/quest=10943/childrens-week
2a.http://www.wowhead.com/quest=10950/auchindoun-and-the-ring-of-observance
2b.http://www.wowhead.com/quest=10954/jheel-is-at-aeris-landing (also Jheel should end this quest, but he doesn't have the "?" over his head)
2c.http://www.wowhead.com/quest=10952/a-trip-to-the-dark-portal
3a.http://www.wowhead.com/quest=10962/time-to-visit-the-caverns
3b.http://www.wowhead.com/quest=10956/the-seat-of-the-naaru
4.http://www.wowhead.com/quest=10966/back-to-the-orphanage
Horde Shatt quests:
1.http://www.wowhead.com/quest=10942/childrens-week
2a.http://www.wowhead.com/quest=10945/hchuu-and-the-mushroom-people
2b.http://www.wowhead.com/quest=10953/visit-the-throne-of-the-elements
2c.http://www.wowhead.com/quest=10951/a-trip-to-the-dark-portal
3a.http://www.wowhead.com/quest=11975/now-when-i-grow-up
3b.http://www.wowhead.com/quest=10963/time-to-visit-the-caverns
4.http://www.wowhead.com/quest=10967/back-to-the-orphanage
Dalaran Oracle quests:
1.http://www.wowhead.com/quest=13926/little-orphan-roo-of-the-oracles
2a.
2b.
2c.   (update in progress)
3.
4.
5.

**Expected behaviour**: Everyone should have the chance to do ALL the orphan quests

**Steps to reproduce the problem**: Try to do any of the quests

**Revision**: Current server version (you can see current server revision by typing ".server info" command in the game chat): 5371

**Comment by Llagas** [comment]
Please fix this before the event ends :)

**Comment by pelekon** [comment]
I will try to fix as many quests as I can till the end of event and I will post it on forum, so pray that Bildo would accept them till the end of event :D

**Comment by Llagas** [comment]
If not all the quests, at least the outland ones (draenei ones) so we can get the "Veteran Nanny" achievement :D

**Comment by 1Andas** [comment]
Thanks for this huge report Llagas. 
I hope this gets fixed before the event expires, or maybe they could prolong it a bit so we can finish once they fix.

**Comment by Sainsyl** [comment]
suprise,suprise, another year no fix for this event...

**Comment by Pieczyyea** [comment]
Another year, still not working

**Comment by Llagas** [comment]
Could you at least try and fix the shattra quests so that we can start getting the veteran nanny achievement :)

**Comment by pelekon** [comment]
3/4 fixed in rev 5749

**Comment by pelekon** [comment]
As I wrote on discord I will try to script them until weekend, sadly I am sick and right now its hard for me to even open core to script it

**Comment by Llagas** [comment]
Its fine don't worry about it, we can wait another year for thoses. 
Just focus on getting well

**Comment by jissabraham** [comment]
not giving [http://www.wowhead.com/quest=10966/back-to-the-orphanage](url) for allys

**Comment by pelekon** [comment]
@jissabraham Did you completed both quests in exodar?

**Comment by jissabraham** [comment]
@pelekon yes i did first at [].http://www.wowhead.com/quest=10950/auchindoun-and-the-ring-of-observance(url) 2.[ http://www.wowhead.com/quest=10954/jheel-is-at-aeris-landing)](url)
and 3 at dark portal and last one with toy after that no more quest is given by kido to finish the chain

**Comment by pelekon** [comment]
I am not sure if you got what I mean :) Are you sure that you did http://www.wowhead.com/quest=10968/call-on-the-farseer ? This quest is given by Naaru in exodar, and it has to be completed to unlock last quest.

**Comment by jissabraham** [comment]
yes that was done too and the kid stopped giving me quest

**Comment by pelekon** [comment]
Idk then, I just did whole chain without problems
